src/stylesheets/components/_form.scss

```diff
@@ -47,3 +47,7 @@ form {
     }
   }
 }
+
+.alert-danger {
```

Comment:
This style doesn't quite belong here. It is not within the `form` style block, which means it will be applied in any component that uses `.alert-danger`. If this style is intended to be applied everywhere, it should be more appropriately placed in it's own file _../alert.scss_

Reply:
Yup, I wanted to have the css applied to the `.alert-danger` element only when its used for a form. However the alert-danger is being mounted outside of the form element, which means that there is no way I can have the css for `.alert-danger` nested within the form style block. I will take your advice and place it in its own file